core/templates/dev/head/pages/collection_editor/CollectionEditor.ts

```diff
@@ -47,9 +48,23 @@ oppia.constant(
 oppia.constant('INTERACTION_SPECS', GLOBALS.INTERACTION_SPECS);
 
 oppia.controller('CollectionEditor', [
-  'CollectionEditorStateService',
-  function(CollectionEditorStateService) {
+  '$scope', 'CollectionEditorStateService', 'PageTitleService',
+  'EVENT_COLLECTION_INITIALIZED', 'EVENT_COLLECTION_REINITIALIZED',
+  function($scope, CollectionEditorStateService, PageTitleService,
+      EVENT_COLLECTION_INITIALIZED, EVENT_COLLECTION_REINITIALIZED) {
     // Load the collection to be edited.
     CollectionEditorStateService.loadCollection(GLOBALS.collectionId);
+    var setTitle = function() {
+      if (CollectionEditorStateService.getCollection().getTitle()) {
+        PageTitleService.setPageTitle(CollectionEditorStateService
+          .getCollection().getTitle() + ' - Oppia Editor');
+      } else {
+        PageTitleService.setPageTitle(
+          'Untitled Collection - Oppia Editor');
```

Comment:
Can we move this to the line above? (Following 80 char max per line)

Reply:
Done